benchexec/tools/tabol.py

```diff
@@ -19,29 +19,27 @@
 import benchexec.tools.template
 
 class Tool(benchexec.tools.template.BaseTool):
+    REQUIRED_PATHS = []
```

Comment:
Could you please add a proper set of required paths? E.g., `tabol.jar`, `tabol.sh` and whatever else is necessary.

This is not necessary for BenchExec itself, but for being able to automatically distribute the tool across various machines.

Reply:
Thanks for your suggestion! Then, I add some requried path and the version number.